src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -128,20 +127,13 @@
 
 (defn- f-view
   []
-  (let [on-close       (fn []
-                         (rf/dispatch [:wallet/clean-scanned-address])
-                         (rf/dispatch [:wallet/clean-local-suggestions])
-                         (rf/dispatch [:wallet/clean-selected-token])
-                         (rf/dispatch [:wallet/clean-selected-collectible])
-                         (rf/dispatch [:wallet/clean-send-address])
-                         (rf/dispatch [:wallet/select-address-tab nil])
-                         (rf/dispatch [:navigate-back]))
+  (let [on-close       #(rf/dispatch [:navigation/wizard-back-send-flow])
         on-change-tab  #(rf/dispatch [:wallet/select-address-tab %])
         input-value    (reagent/atom "")
         input-focused? (reagent/atom false)]
     (fn []
       (let [selected-tab          (or (rf/sub [:wallet/send-tab]) (:id (first tabs-data)))
-            token                 (rf/sub [:wallet/wallet-send-token])
+            ;; token                 (rf/sub [:wallet/wallet-send-token])
```

Comment:
why do we comment this btw?

Reply:
It was not used in the code. I forgot to remove the line